resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver/Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM ubuntu:trusty
+FROM anapsix/alpine-java:8
 
 # Upgrade package index
-# install a few other useful packages plus Open Jdk 7
 # Remove unneeded /var/lib/apt/lists/* after install to reduce the
 # docker image size (by ~30MB)
-RUN apt-get update && \
-    apt-get install -y less openjdk-7-jre-headless net-tools vim-tiny sudo openssh-server procps && \
-    rm -rf /var/lib/apt/lists/*
+RUN apk add --update vim
```

Comment:
There's no `vi` by default?  If so I'd want to keep this slimmer by not including `vim`